### DISCUSSION TITLE
Multiselect keys

### DISCUSSION THREAD

**Discussion by salahaldin** [primefaces/community#1030] [PrimeReact] [answered]
Hi, 
I am having a table and a multi select to set the visible columns.  If I remove the multiSelect all looks good, the moment I add the multi select I get 

`react-dom.development.js:102 Warning: Encountered two children with the same key, `0`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version`

How to set the keys for DataTable and MultiSelect to solve such warnings.  I had a look on multi select and I was not able to find how to assign a key. 

Thanks in advance

**Comment by melloware** [comment, marked as answer]
its `dataKey` property: https://primereact.org/multiselect/#api.MultiSelect.props.dataKey

**Reply by salahaldin** [reply]
Thank you for quick response works as charm
